New published image fossology/fossology:scanner doesnt work

<!-- SPDX-FileCopyrightText: © Fossology contributors

     SPDX-License-Identifier: GPL-2.0-only
-->

<!-- Before filling this issue, please read the Wiki (https://github.com/fossology/fossology/wiki)
and search if the bug do not already exists in the issues (https://github.com/fossology/fossology/issues). -->

### Description

Latest fossology/fossology:scanner has a problem, it needs ScanDeps module in python env.

#### How to reproduce

For a bug: Describe the bug and list the steps you used when the issue occurred.
all have same errors, just sharing one of the command.
```
docker run --rm --name "fossologyscanner" \
            -t -w "/opt/repo" \
            -v ${PWD}:/opt/repo \
            fossology/fossology:scanner \
            "/bin/fossologyscanner" repo nomos ojo
```

#### Screenshots

If applicable, add screenshots to help explain your problem.

### Versions

* Last commit id on master:
Container version
index digest: sha256:4cc8cee4c0219f9e55b615f599fa0d08a7108b5288d764d934ad5223e233cfd5
manifest digest: sha256:f56bf3f73e60384640fccd4ec9d6d029028392cf76ba641d5df48f7f63151cdf
### Logs

Any logs (if any) generated in
```
Traceback (most recent call last):
  File "/bin/fossologyscanner", line 24, in <module>
    from ScanDeps.Parsers import Parser, PythonParser
ModuleNotFoundError: No module named 'ScanDeps'
```
it works with older version;
`fossology/fossology:scanner@sha256:8269d1f37b65f01421d9a6eb53002e320c5ee8e2c84c8caca8039cbbfd8b1105
`

I believe i've got the same issue.

My github action - a minimal version of the [the github actions sample](https://github.com/fossology/fossology/blob/master/utils/automation/.github-workflow.yml) - fails to run:

**Stacktrace**
```
Traceback (most recent call last):
  File "/bin/fossologyscanner", line 24, in <module>
    from ScanDeps.Parsers import Parser, PythonParser
ModuleNotFoundError: No module named 'ScanDeps'
Error: Process completed with exit code 1.
```

**Github Actions Yaml**
```
name: Check diff for license with Fossology

on:
  push:
    branches:
      - me/fossology

jobs:
  check-license:
    name: Check license
    runs-on: ubuntu-latest
    steps:
      - name: Checkout this repository
        uses: actions/checkout@v3

      - name: Set up QEMU
        uses: docker/setup-qemu-action@v3

      - name: Run Scanners
        run: |
          docker run --rm --name "fossologyscanner" -w "/opt/repo" -v ${PWD}:/opt/repo \
            -e GITHUB_TOKEN=${{ github.token }} \
            -e GITHUB_PULL_REQUEST=${{ github.event.number }} \
            -e GITHUB_REPOSITORY=${{ github.repository }} \
            -e GITHUB_API=${{ github.api_url }} \
            -e GITHUB_REPO_URL=${{ github.repositoryUrl }} \
            -e GITHUB_REPO_OWNER=${{ github.repository_owner }} \
            -e GITHUB_ACTIONS \
            fossology/fossology:scanner "/bin/fossologyscanner" nomos ojo
```

Please note: This is my first try at using Fossology, so i might have done something wrong entirely

--- a/utils/automation/Dockerfile.ci
+++ b/utils/automation/Dockerfile.ci
@@ -71,6 +71,7 @@ COPY --from=builder /fossology/run-deps /opt/run-deps
 
 COPY ./utils/automation/fossologyscanner.py /bin/fossologyscanner
 COPY ./utils/automation/FoScanner /bin/FoScanner
+COPY ./utils/automation/ScanDeps /bin/ScanDeps
 COPY ./utils/automation/requirements.txt /bin/
 
 RUN echo 'APT::Install-Recommends "0" ; APT::Install-Suggests "0" ;' >> /etc/apt/apt.conf \

--- a/utils/automation/fossologyscanner.py
+++ b/utils/automation/fossologyscanner.py
@@ -351,12 +351,16 @@ def main(parsed_args):
     else:
       print(f"Could not validate keyword file: {message}")   
 
+  valid_comps_exist = False
   if (cli_options.scan_only_deps or cli_options.repo) and cli_options.sbom_path != '':
     download_list = []
     save_dir = 'pkg_downloads'
     sbom_file_path = cli_options.sbom_path
     parser = Parser(sbom_file_path)
     parser.classify_components()
+    valid_comps_exist = ( parser.python_components != [] or
+                        parser.php_components != [] or
+                        parser.npm_components != [] )
 
     python_comps = parser.python_components
     unsupported_comps = parser.unsupported_components
@@ -380,9 +384,6 @@ def main(parsed_args):
   if cli_options.repo is False:
     cli_options.diff_dir = repo_setup.get_diff_dir()
 
-    valid_comps_exist = ( parser.python_components != [] or
-                  parser.php_components != [] or
-                  parser.npm_components != [] )
     if cli_options.scan_only_deps and valid_comps_exist:
       cli_options.diff_dir = save_dir
     if cli_options.scan_dir:

--- a/utils/automation/requirements.txt
+++ b/utils/automation/requirements.txt
@@ -4,3 +4,4 @@
 # SPDX-License-Identifier: FSFAP
 # Description: Dependencies used by the CI scanner script
 spdx-tools==0.8.2
+requests==2.25.1

